packages/scripts/config/webpack.config.js

```diff
@@ -260,6 +263,26 @@ if ( baseConfig.devtool ) {
 	} );
 }
 
+/**
+ * Build blocks manifest.
+ */
+class BlocksManifestPlugin {
+	/**
+	 * Apply the plugin.
+	 *
+	 * @param {webpack.Compiler} compiler The compiler instance.
+	 */
+	apply( compiler ) {
+		compiler.hooks.afterEmit.tap( 'BlocksManifest', () => {
+			exec(
+				`node ${ fromScriptsRoot(
```

Comment:
I changed here two things in [d01a653](https://github.com/WordPress/gutenberg/pull/69578/commits/d01a653b1792be97366b8c7c6500fb884500cd07).
1. Use the local script rather than `wp-scripts` executable so it can run locally in non-standard way. The way I usually test changes is by running `../node_modules/.bin/wp-scripts start --blocks-manifest --output-path=dist` from the scaffolded block without the npm packages installed. This way I can use the latest version of `wp-scripts` from the Gutenberg repository.
2. I added a way to change the `--input`  param in the case when someone decides the build folder should live in a different location.

Reply:
Thanks for these fixes!